### ISSUE TITLE
🐛 Bug Report — Runtime APIs - fts5 allowed but not fts5vocab

### ISSUE THREAD

**Issue by rozenmd** [cloudflare/workerd#1540]
> The fts5vocab virtual table module allows users to extract information from an FTS5 full-text index directly. The fts5vocab module is a part of FTS5 - it is available whenever FTS5 is.

from https://www.sqlite.org/fts5.html

attempting to use fts5vocab in a query throws an error:

```
> CREATE VIRTUAL TABLE ft1_v USING fts5vocab('ft1', 'Order');
not authorized
```

Issue originally raised here: https://github.com/cloudflare/workers-sdk/issues/4543

**Comment by mhart** [comment]
Fixed in #2259

**Comment by naourass** [comment]
Was this fix published? I'm trying to create an `fts5vocab` table using a migration file but getting the `not authorized` error even when trying to apply the migration locally:
```
CREATE VIRTUAL TABLE articles_fts_vocab USING fts5vocab(articles_fts, 'row');
```
Error after wrangler migration command:
```
❌ Migration xxxxxxx.sql failed with the following errors:
✘ [ERROR] not authorized
```
